### PR TITLE
Added more useful theorems about UpWords

### PR DESCRIPTION
**Pull request by ProgramCrafter**
Used and promoted four mathbox theorems:
1. `bj-issetiv` -> `issetiv`. An element of a class exists, inference version, by BJ.
2. `leltletr`. Weak transitive law for ordering on reals, by AV.
3. `fzindd`. Induction on the integers from M to N inclusive, a deduction version, by metakunt.
4. `elfzop1le2`. A member in a half-open integer interval plus 1 is less than or equal to the upper bound, by Glauco Siliprandi.

I believe I've found a suitable section for each of them so that the theorems would thematically fit with surrounding ones.

---
Also, Metamath-exe somewhy refused to rewrap theorems' hypotheses; I wrapped those manually, we shall see if I did that correctly...

### CODE REVIEW

**Review by avekens — CHANGES_REQUESTED** [review]
see inline comments

File: set.mm [hunk 1]
```diff
@@ -105,6 +105,7 @@ been continuously enriched since then (list of contributors below). $)
 JY  Jonathan Yan
 FZ  Fan Zheng
 KZ  Kunhao Zheng
+ET  Ender Ting
```
[the remark this answers is not in the document]
Comment (ProgramCrafter) [reply set.mm]: Done!

**Review by icecream17 — APPROVED** [review]
add
```
22-Nov-24 bj-issetiv issetiv    moved from BJ's mathbox to main set.mm
```
to `changes-set.txt`

example: https://github.com/metamath/set.mm/commit/1843f4e0b5ab4a6d9eacb623d194cee1fa58aa4b

----
Note: I'm not sure on how bj-isseti should be handled, so I think the comment of bj-isseti can stand, and a future edit could be combined with a shortening of isseti. Though note bj-isseti does not use extra axioms (only + df-clab)

It might be useful in the future to prove isupword a la https://us.metamath.org/mpeuni/isgrp.html

**Comment by icecream17** [comment]
@avekens the inline comments do not seem to be published

(oops I probably duplicated the changes-set change, oh well)

**Comment by avekens** [comment]
> @avekens the inline comments do not seem to be published


Sorry, I think I forgot to push the "Add comment" button. My inline comment is available now.

**Review by avekens — CHANGES_REQUESTED** [review]
My request to push back ~issetiv (as ~bj-issetiv) into the BJ's mathbox, and to use ~isseti in the proof of ~tworepnotupword instead is not taken into account yet.

File: set.mm [hunk 2]
```diff
@@ -26106,6 +26107,23 @@ the definition of class equality ( ~ df-cleq ).  Its forward implication
...
+    issetiv.1 $e |- A e. V $.
+    $( An element of a class exists, inference version.  Version of ~ bj-isseti
+       with a disjoint variable condition on ` x , V ` .  The hypothesis uses
+       ` V ` instead of ` _V ` for extra generality.  This is indeed more
+       general than ~ isseti as long as ~ elex is not available (and the
+       non-dependence of ~ issetiv on special properties of the universal class
+       ` _V ` is obvious).  Prefer its use over ~ bj-isseti when sufficient (in
+       particular when ` V ` is substituted for ` _V ` ).  (Contributed by BJ,
+       14-Sep-2019.)  (Proof modification is discouraged.) $)
+    issetiv $p |- E. x x = A $=
```
[the remark this answers is not in the document]
Comment (ProgramCrafter) [reply set.mm:26121]: Alright now I see this comment and shall do.

**Comment by ProgramCrafter** [comment]
I've read that there is no theorem `|- -. A < A` (ltneverrefl) for a reason that subsequent theorems should specify their domain. Is naming it to "et-ltneverrefl" and adding a "(New usage is discouraged.)" tag a workable course of action?

**Comment by icecream17** [comment]
yes (rename is optional) (If renaming, note that ltneverrefl is inconsistent with ~ ltnri. Admittedly, ltnri is a relatively incomprehensible label, so this is also optional)

**Comment by ProgramCrafter** [comment]
More and more to know... that's interesting, actually)
While we are at it: how to define TeX rendering for my `UpWord` class? Right now, `metamath.tirix.org` only shows "Could not format assertion : No typesetting found for |- (/) e. UpWord S with typecode |-".

**Review by avekens — APPROVED** [review]
Fine for me now.

**Comment by GinoGiotto** [comment]
> While we are at it: how to define TeX rendering for my `UpWord` class? Right now, `metamath.tirix.org` only shows "Could not format assertion : No typesetting found for |- (/) e. UpWord S with typecode |-".

If I remember correctly, `metamath.tirix.org` rielaborates the typesetting defined from the database, which you already added in the previous PR:

<pre>
/* Mathbox of Ender Ting */
htmldef "UpWord" as
    "UpWord";
  althtmldef "UpWord" as "UpWord";
  latexdef "UpWord" as "\mathrm{UpWord}";
/* End of Ender Ting's mathbox */
</pre>

If this doesn't work, this might indicate a bug in https://github.com/tirix/metamath-web. @tirix 

A similar issue [was present for vx](https://github.com/tirix/metamath-web/issues/16#issuecomment-1784939977), but that has been fixed.

**Comment by avekens** [comment]
> More and more to know... that's interesting, actually) While we are at it: how to define TeX rendering for my `UpWord` class? Right now, `metamath.tirix.org` only shows "Could not format assertion : No typesetting found for |- (/) e. UpWord S with typecode |-".

The typesetting definitions for UpWord seem to be OK:

```
htmldef "UpWord" as "UpWord";
  althtmldef "UpWord" as "UpWord";
  latexdef "UpWord" as "\mathrm{UpWord}";
```

The message complains about "|-", doesn't it? That's strange. @tirix do yo have any idea?

By the way, a space character at the end would result in a nicer HTML represenation (currently "⊢ ∅ ∈ UpWord𝑆"):

```
htmldef "UpWord" as "UpWord ";
  althtmldef "UpWord" as "UpWord ";
  latexdef "UpWord" as "\mathrm{UpWord}";
```
would result in "⊢ ∅ ∈ UpWord 𝑆"

Compare with the typesetting for Slot ("class Slot 𝐴"):
```
htmldef "Slot" as "Slot ";
  althtmldef "Slot" as "Slot ";
  latexdef "Slot" as "\mathrm{Slot}";
```

**Comment by tirix** [comment]
> > More and more to know... that's interesting, actually) While we are at it: how to define TeX rendering for my `UpWord` class? Right now, `metamath.tirix.org` only shows "Could not format assertion : No typesetting found for |- (/) e. UpWord S with typecode |-".
> 
> The message complains about "|-", doesn't it? That's strange. @tirix do yo have any idea?
> 
The formatting information for [metamath.tirix.org](https://metamath.tirix.org/mpests/upwordnul) is not provided in the main `set.mm` file, but in [`set-mathml.mmts`](https://github.com/metamath/set.mm/blob/develop/set-mathml.mmts). The error warns that typesetting information is missing for the whole expression, actually. Maybe it could be phrased a bit better. It should be easy to add a new symbol by mimicking other entries there.

That file does not provide translations math token by math token (i.e. for just `UpWord`), but for syntactic expressions instead (i.e. for `class UpWord`).

I'm not keeping up regularly with new math syntax, but instead updating the file in batch from time to time. One way to ensure that it's up to date would be to add a check to the CI. Our CI is already quite strict, but on the other hand it includes some TeX checks which might be less used than [metamath.tirix.org](https://metamath.tirix.org)?

(BTW I see the site SSL certificate has expired, I'll see to renew it!)

**Comment by GinoGiotto** [comment]
> One way to ensure that it's up to date would be to add a check to the CI. Our CI is already quite strict, but on the other hand it includes some TeX checks which might be less used than [metamath.tirix.org](https://metamath.tirix.org/)?

Adding CI might be worth it. I visit [metamath.tirix.org](https://metamath.tirix.org/mpests/upwordnul) regularly and I would like to see the website become more relevant in the future. I believe the modern design and TeX rendering have the potential to attract more people that are curious about metamath.

**Comment by digama0** [comment]
I do think that the no-js restriction is too limiting for metamath.org. I've been trying to move the website to something more modern for ages but it's difficult to get people to thumbs up a plan; or at least that was my impression last time we [discussed it](https://groups.google.com/g/metamath/c/untRQm4N-BI/m/jKrTVEc_AgAJ) on the mailing list.

**Comment by tirix** [comment]
> Adding CI might be worth it. I visit [metamath.tirix.org](https://metamath.tirix.org/mpests/upwordnul) regularly and I would like to see the website become more relevant in the future. I believe the modern design and TeX rendering have the potential to attract more people that are curious about metamath.

OK, I should be able to propose a CI change for that.

**Comment by david-a-wheeler** [comment]
Just to be clear, I never said "no-JS" in 2023. What I said was:

> What I want is for *a* way for people to access the basic information without *requiring* client-side JavaScript. I see no reason that it be the *only* way we provide the information, we already provide it in multiple forms. I just want to retain that capability. So you don't need to feel held back :-).
> ...
> As far as speed goes, JavaScript is often slower than HTML where both can be used. In practice JavaScript files are often quite large, and JavaScript has to be parsed, examined, executed, and then its execution must cause document tree changes which finally get rendered. Web browsers are *really* good at rapidly & progressively rendering HTML. On systems I've worked with, HTML (even with server roundtrips) is often *MUCH* faster when measuring paint-to-screen times. JavaScript *has* gotten faster over the years, thankfully, but the gap is still large. Of course, when you need functionalities only provided by JavaScript then you need it. Like all technologies there are tradeoffs.

However, as this isn't really about UpWords, this topic should be discussed in its own issue or the mailing list.

**Comment by ProgramCrafter** [comment]
Hello! Just a reminder: I think this PR is ready for merging; style changes for "UpWord " and etc would certainly belong to next ones.

**Comment by avekens** [comment]
@ProgramCrafter please move you mathbox between "Mathbox for Saveliy Skresanov" and "Mathbox for Jarvin Udandy" with your next PR, bacause we order the mathbox alphabetically (by surname). So the contents should look like the following:

      20.38  Mathbox for Saveliy Skresanov
      20.39  Mathbox for Ender Ting
      20.40  Mathbox for Jarvin Udandy
      20.41  Mathbox for Adhemar (???)
      20.42  Mathbox for Alexander van der Vekens
      20.43  Mathbox for Zhi Wang
      20.44  Mathbox for Emmett Weisz
      20.45  Mathbox for David A. Wheeler
      20.46  Mathbox for Kunhao Zheng

**Pull request merged**